raiden/tests/utils/network.py

```diff
@@ -30,6 +31,19 @@
 )
 
 
+def dict_merge(to_update, other_dict):
```

Comment:
We already have a utils for that:
https://github.com/raiden-network/raiden/blob/ae3c18297b0f166e8ceb016ef48550090a8a8b8e/raiden/utils/__init__.py#L297-L311